### ISSUE TITLE
awesome Wiki

### ISSUE THREAD

**Issue by hi5** [ahkscript/awesome-AutoHotkey#92]
Should we turn on the Wiki for awesome-AutoHotkey so we can add other content which may not fit directly in the awesome format.

Examples

- [AutoHotkey "typing" related scripts](http://www.autohotkey.net/~faqbot/typing-scripts.html)
- A page about "virus warnings" [ref](https://autohotkey.com/boards/viewtopic.php?f=14&t=15062)
- A page about "installed AutoHotkey but it doesn't work" - several on the forum past few days.

Advantage: anybody can update wiki as opposed to a forum post which newbies won't find anyway

Just thought I'd put it up for discussion.

**Comment by hoppfrosch** [comment]
Any script, which shows some great features of Autohotkey - as most scripts from your link [AutoHotkey "typing" related scripts](http://www.autohotkey.net/~faqbot/typing-scripts.html) might do, should be worth to be added to the awesome list directly. As we now TravisCI-Integration with link validation for our awesome list, it's easy for our list to identify invalid entries - relying on other lists does not ensure the entries of the other lists are still available - also therefore any "worthy" script should be added directly to our list.

I think it would be great to have a place to collect hints for newbies to solve problems they have with AutoHotkey (virus warnings, "won't start") - but I'm not quite sure this should be part of the awesome list ....

**Comment by vasili111** [comment]
I agree with @hoppfrosch . I think that scripts that are useful for other people should be in awesome list. I dont think it is good idea to have wiki and list for nearly same purpose.
For useful tips tricks and FAQ there can be made separate repo for it and I think it will be useful.

**Comment by hoppfrosch** [comment]
As nobody else voted positively yet, I've marked it as a candidate for the sack. I'll leave it in this state for a few weeks - if there's no reaction, I'll consider this item as rejected

**Comment by joedf** [comment]
I just wish the original AHK wiki wasn't screwed up. :/

**Comment by hoppfrosch** [comment]
Same here ...
If the "awesome Wiki" isn't meant as a duplicate of the awesome list but rather  a replacement of the old AHK wiki, we, the name should be changed to AHK wiki - and should establish a new one ;-)

**Comment by joedf** [comment]
+1
